readme.md

```diff
@@ -148,6 +148,7 @@ Made with Electron.
 - [Wire](https://github.com/wireapp/wire-desktop) - Messenger and calling app.
 - [Ramme](https://github.com/terkelg/ramme) - Unofficial Instagram app.
 - [Moonview](https://github.com/teesloane/moonview) - Ambient notepad.
+- [Temps](https://jackd248.github.io/temps/) - A simple but smart weather app.
```

Comment:
"Smart" means for me and for Temps, that it is innovative, modern and informative but not intrusive or overloaded. If this is not exact enough for you, "simple weather app" is also working for me.